### DISCUSSION TITLE
How to get Jpeg2000 image 's  image.Quality

### DISCUSSION THREAD

**Discussion by Charltsing** [dlemstra/Magick.NET#1490] [Help]
### Magick.NET version

13.3.0.0

### Environment (Operating system, version and so on)

Windows10, net framework 4.7.2

### Description

image.Quality changed from 0 to 1 when i set  image.Quality = 0
Can I set this minimum value ? 
For example, if quality=0, it defaults to 35 （This minimum value can constrain all image formats, such as jpg, jp2...)
I think it's meaningless to set quality=1 for image.

[1.zip](https://github.com/dlemstra/Magick.NET/files/13696647/1.zip)

I want to know ， **how to get image.Quality when load jpeg2000 image.**
i found  image.Quality is 0 when load 3.jp2 ( it's inside the zip file ) ,
but when i save it use  image.Quality = 100, it filesize is 1.63M (24bit)
when i save it use  image.Quality = 30, it filesize is 328k

I want to get the value of image quility when loading the jpeg2000 image,
so I can make sure the file size doesn't change too much when saving the same jpeg2000 image.

### Steps to Reproduce

image.Quality = projectimageinfo.OriQuality;

look at the 1.gif file inside the zip file

**Comment by dlemstra** [comment]
There is no way to read the quality from a jp2 image.
